paddle/framework/executor.h

```diff
@@ -45,5 +45,7 @@ class Executor {
   const platform::Place place_;
 };
 
+static void CreateTensor(Variable* var, proto::VarDesc::VarType var_type);
```

Comment:
Why need to be `static`?

Reply:
Will make it as a `global function` and I think it's a `static global function` in the previous just because the function doesn't need to export for other files(only `executor.cc` need it).

Reply:
Done.